.github/ISSUE_TEMPLATE/contribute.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Contribute
+about: Let us know you have an example you'd like to contribute
+
+---
+
+**Description**
+
+A clear and concise description of the concepts in your project and the problem 
+your code solves. Please include any references you used. 
+
+**How to run the example**
+
+A short description of how to run your code, including an example. 
+
+**Link to your repo**
+
+Include a link to the repo you want to contribute to d-wave examples:
+
+https://github.com/dwave-examples
+
+**Method of contributing**
+
+There are two options for how to contribute your example:
+1. Forking  
+    D-Wave will fork your repo. We may make a pull request to your repo if we would 
+	like your approval for certain code changes.
+2. Transferring repo ownership
+    You may choose to transfer ownership of your repo to D-Wave. Here's more 
+	information on this option: 
+	
+	https://help.github.com/en/github/administering-a-repository/transferring-a-repository
```

Comment:
In the collection of examples, will this repo show up as a "D-Wave" example or an external contribution? I'm not sure how that differentiation occurs in the metadata-grabbing code. @randomir

Reply:
Good point. It'll show up as owned by D-Wave. Right now, I'm not sure we can differentiate it on the examples page, but I'll double-check when you test out the transfer.